cmd/kube-controller-manager/app/plugins.go

```diff
@@ -31,13 +31,25 @@ import (
 )
 
 // ProbeRecyclableVolumePlugins collects all persistent volume plugins into an easy to use list.
-func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
+func ProbeRecyclableVolumePlugins(config VolumeConfig) []volume.VolumePlugin {
```

Comment:
This isn't new.  It already exists.  Its the func used by the binary to load the plugins.